### PR TITLE
Updated to indicate M1 with Rosetta is supported.

### PR DESCRIPTION
**Pull request by vazquea**

### CODE REVIEW

**Comment by toolkitty** [comment]
[Documentation Preview](http://sg-devdocs.s3-website-us-east-1.amazonaws.com/tk-doc-generator/e0d64b4704cd0eabf167e81ca2dba3ccb47a674e/index.html)

**Review by mathurf — APPROVED** [review]
LGTM!
Sorry for the delay I didn't notice this review request

File: docs/en/reference/pipeline-integrations/toolkit-engines-supported-os-versions.md [hunk 1]
```diff
@@ -13,4 +13,4 @@ Here is a list of Toolkit engine supported operating systems.
 
 | Windows | macOS | CentOS |
 |:-------:|:-----:|:------:|
-|   10    |10.14+ <br>**Note:** M1 is not supported yet, [learn more here](https://community.shotgridsoftware.com/t/13606).    |  7.4   |
\ No newline at end of file
+|   10    |10.14+ <br>**Note:** M1 native is not supported, M1 with Rosetta is supported.    |  7.4   |
```
Comment (mathurf) [inline docs/en/reference/pipeline-integrations/toolkit-engines-supported-os-versions.md:16]: One question... why you removed link to community post about how to enable Rosetta in Photoshop?

**Review by vazquea — COMMENTED** [review]
Restored link for PS workaround.

**Comment by toolkitty** [comment]
[Documentation Preview](http://sg-devdocs.s3-website-us-east-1.amazonaws.com/tk-doc-generator/659bb1be29fdb755008f33d39cb75edf809ca5c6/index.html)

**Pull request merged**